main.go

```diff
@@ -181,10 +181,10 @@ func main() {
 		if err != nil {
 			logger.Fatalf(err.Error())
 		}
-		if err := client.WaitForTaskNotification(ctx); err != nil {
-			logger.Fatalf(err.Error())
+		for {
+			client.WaitForTaskNotification(ctx)
+			time.Sleep(10 * time.Minute)
```

Comment:
Under what conditions can we end up in this state?

Reply:
This is just for running waitfortasknotification on the command line, and it mirrors what we have with the service. WaitForTaskNotification is reconfigured every 10 min. For the most part this is a noop. The expectation is that if you want to quit you just ctrl-c.

Reply:
Changed to a select so this is clearer